component/main.jsonnet

```
@@ -84,4 +84,5 @@ local imageConfigSpec =
     tls.secrets,
   [if std.length(tls.certs) > 0 then '30_cert_manager_certs']:
     tls.certs,
+  '20_monitoring': (import 'monitoring.libsonnet'),
 }
```

It's cleaner if we do the following here, as we don't create an empty file that way if monitoring isn't enabled:

```suggestion
} +   (import 'monitoring.libsonnet')
```